Dockerfile

```diff
@@ -3,6 +3,8 @@
 # https://catalog.redhat.com/software/containers/ubi9/go-toolset/61e5c00b4ec9945c18787690
 FROM registry.access.redhat.com/ubi9/go-toolset:1.22.9-1737480393 AS builder
 
+USER root
```

Comment:
Why was the `USER root` needed?

Reply:
@chmeliik sealights generates the packages with a different user when https://github.com/konflux-ci/build-definitions/blob/main/task/sealights-go-oci-ta/0.1/sealights-go-oci-ta.yaml#L139-L140. Not sure how to mitigate this. If you see a solution im open

Reply:
Ok, this should be fine. The final stage of the Dockerfile uses a non-root user.